manifests/server/export.pp

```diff
@@ -57,7 +57,9 @@
 # @param v4_export_name
 # @param nfsv4_bindmount_enable
 #
-# @examples
```

Comment:
please re-add the `@examples` strings tag.

Reply:
I think it should be `@example`, and it's is a tag used by puppet strings to render an example in REFERANCE.md
See https://www.puppet.com/docs/puppet/8/puppet_strings.html

It should be moved below the last `@param` at line 65

Reply:
Should I create a separate pull request of off the current master to fix @examples/@example?  

If so, #193

Reply:
No need to have another PR. please fix it here and please keep in mind that the code example must be indeted by 2 spaces.

Reply:
I've fixed it here but I have no idea what you're referring to when you say, "...code example must be indented by 2 spaces."

Reply:
@bschonec the example should have a description, then the example code. The code should be indented two spaces, so there should be three spaces after the hash mark `#`. Examples: https://github.com/search?q=repo%3Avoxpupuli%2Fpuppet-systemd%20%40example&type=code

Reply:
Tag was readded. This is OK for me. We can solve this conversation.